scenariorun/src/main/openapi/scenariorun.yaml

```diff
@@ -337,6 +337,41 @@ paths:
               examples:
                 ScenarioRun:
                   $ref: '#/components/examples/BreweryScenarioRun'
+  /organizations/{organization_id}/scenarioruns/{scenariorun_id}/workflows/{workflow_name}/stop:
```

Comment:
You don't need to set workflow_name in path, it is stored in the scenariorun definition:
/organizations/{organization_id}/workspaces/{workspace_id}/scenarioruns/{scenariorun_id}/stop